docs/configuring-kubelinter.md

```diff
@@ -46,6 +46,15 @@ checks:
   addAllBuiltIn: true
 ```
 
+## Ignore paths
+
+To ignore checks on files or direcotry under certain paths, add ignored paths to `ignorePaths`.
```

Comment:
Do we support only absolute paths?

Reply:
How about supporting globs?

Reply:
Relative path is supported as well, and added glob path support. Updated doc in [03c80d1](https://github.com/stackrox/kube-linter/pull/353/commits/03c80d17f4c666035f37ec3d2e6a54fc4bb8fe7d)